packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/app-image.html

```diff
@@ -74,16 +147,12 @@
                 ></li>
                 <li
                     data-translate="pci_app_submit_select_image_info_message_2"
-                    data-translate-values="{ 'architectures': '<strong>(linux/AMD64)</strong>' }"
+                    data-translate-values="{ 'architectures': '<strong>	&lsquo;linux/amd64&rsquo;</strong>' }"
```

Comment:
**nitpick:** This can be maintained as a constant in the constant file

Reply:
Done: https://github.com/ovh/manager/pull/9413/commits/8f753a833c9a5a9857c174fe32824151271b5412